www/js/splash/notifScheduler.js

```diff
@@ -65,6 +65,14 @@ angular.module('emission.splash.notifscheduler',
             if (!notifs?.length)
                 return Logger.log(`${prefix}, there are no scheduled notifications`);
             const time = moment(notifs?.[0].trigger.at).format('HH:mm');
+            cordova.plugins.notification.local.scheduledNotifs = notifs.map((n) => {
```

Comment:
I don't think we should put this into the plugin - the plugin is typically read-only in javascript. This should go into the rootScope or a service

Reply:
Ah, this was my fault, not @Abby-Wheelis

Yes, `$rootScope`, or a variable at the root level of `scheduler`, would be a better choice